examples/qml_features/rust/src/custom_base_class.rs

```diff
@@ -34,7 +34,9 @@ pub mod qobject {
     // ANCHOR: book_inherit_qalm
     // ANCHOR: book_qobject_base
     extern "RustQt" {
-        #[cxx_qt::qobject(base = "QAbstractListModel", qml_element)]
```

Comment:
assume we think that below is a better syntax ?

Or do we think that `#[qobject(base = "QAbstractListModel", qml_element)]` is better, but i think this is harder to do with syn only methods rather than our custom parsing that was removed in this change :-) 

The new syntax below isn't far off what you'd do in C++ anyway though as for that each of them is a separate macro
```cpp
class CustomBaseClass : public QAbstractListModel {
    Q_OBJECT
    QML_ELEMENT
    ...
}
```

Reply:
It also matches CXX well with
```rust
#[namespace = "A"]
#[cxx_name = "B"]
#[rust_name = "C"]
type D = ...;
```

(and we already support `namespace` here, cxx_name / rust_name would be a further extension i'll add as another issue)

Reply:
Done as #635

Reply:
I think for continuity with Qt it's nice to have these as separate attributes.
That would also allow us to automatically apply `#[qobject]` to all types in a `RustQt` bridge, as that would then just mean removing the `#[qobject]` attribute.